packages/lab/src/date-picker/DatePicker.tsx

```diff
@@ -103,6 +103,7 @@ export interface DatePickerProps
    * Callback fired when the input value change.
    */
   onChange?: ChangeEventHandler<HTMLInputElement>;
+  isCompact?: boolean;
```

Comment:
we might have to call this `compact`, let me double check on the naming

Reply:
let's rename it, it is already used as `compact` in grid so it would keep it consistent :)

Reply:
done